samples/ConsoleApp/ConsoleApp.csproj

```diff
@@ -15,7 +15,10 @@
 
     <!-- Hack to fix SDK dependencies -->
     <PackageReference Include="System.Diagnostics.DiagnosticSource" Version="6.0.0" />
-    <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="2.1.0" />
+    <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="3.0.3" />
+
+    <!-- Hack to make the otel exportert working for .NET 5.0 -->
+    <PackageReference Include="Grpc.Net.Client" Version="2.32.0" Condition="'$(TargetFramework)' == 'net5.0'"/>
```

Comment:
If we want this merged then it should be documented in `USAGE.md`

Reply:
FYI https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/350